recipe/conda_build_config.yaml

```
@@ -20,22 +20,22 @@ cxx_compiler:
   - clangxx                    # [osx]
   - vs2017                     # [win]
 cxx_compiler_version:          # [unix]
-  - 11                         # [osx]
-  - 9                          # [linux]
+  - 12                         # [osx]
+  - 10                         # [linux]
   - 7                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
   - 9                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
   - 9                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
   - 9                          # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
```

For illustrative purpose that would mean updating the last 2 in all of these listings to 10 IIUC

```suggestion
  - 10                         # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
  - 10                         # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux64]
```